app/assets/config/manifest.js

```diff
@@ -1,6 +1,5 @@
 //= link_tree ../images
 //= link_tree ../builds
 //= link_tree ../../javascript .js
-//= link_tree ../../../vendor/javascript .js
-//= link_tree ../../../node_modules/govuk-frontend/govuk/assets
-//= link_tree ../../../node_modules/@ministryofjustice/frontend/moj/assets
+//= link_tree ../../../node_modules/govuk-frontend/dist/govuk/assets
+//= link_tree ../../../node_modules/@ministryofjustice/frontend/moj/assets
```

Comment:
nit - linebreak . what editor do you have? as the `.editorconfig` should do this automatically on save 

https://editorconfig.org/

Reply:
Ta. I've previously relied on rubocop to fix these for me (and just been a bit sloppy about non-ruby files). I've now set up my editor (VS Code) to use the editorconfig file.